tests/test_cipher_configs.py

```diff
@@ -354,23 +385,41 @@ class LargeSIVKeyCipher(control_cipher.__class__):
         assert plaintext == cipher.bytes_decrypt(ciphertext, aad=aad)
 
         enc_stream = await cipher.astream_encrypt(salt=salt, aad=aad)
-        ciphertext = b"".join([b"".join(id_ct) async for id_ct in await enc_stream.abuffer(plaintext)])
-        ciphertext += b"".join([b"".join(id_ct) async for id_ct in enc_stream.afinalize()])
-        dec_stream = await cipher.astream_decrypt(salt=salt, aad=aad, iv=enc_stream.iv)
-        result = b"".join([block async for block in await dec_stream.abuffer(ciphertext)])
-        result += b"".join([block async for block in dec_stream.afinalize()])
+        ciphertext = b"".join(
+            [
+                b"".join(id_ct)
+                async for id_ct in await enc_stream.abuffer(plaintext)
+            ]
+        )
+        ciphertext += b"".join(
+            [b"".join(id_ct) async for id_ct in enc_stream.afinalize()]
+        )
+        dec_stream = await cipher.astream_decrypt(
+            salt=salt, aad=aad, iv=enc_stream.iv
+        )
+        result = b"".join(
+            [block async for block in await dec_stream.abuffer(ciphertext)]
+        )
+        result += b"".join(
+            [block async for block in dec_stream.afinalize()]
+        )
```

Comment:
This section could be more readable. Perhaps turn formatting off for this block.

Reply:
Resolved by (https://github.com/rmlibre/aiootp/pull/12/commits/9c17192b8465434eac060c506f50ab6a8996a1a2)